src/main/scala/chisel3/stage/phases/MaybeFirrtlStage.scala

```diff
@@ -0,0 +1,20 @@
+// See LICENSE for license details.
+
+package chisel3.stage.phases
+
+import chisel3.stage.NoRunFirrtlCompilerAnnotation
+
+import firrtl.AnnotationSeq
+import firrtl.options.Phase
+import firrtl.stage.FirrtlStage
+
+/** Run [[firrt.stage.FirrtlStage FirrtlStage]] if a [[firrtl.stage.NoRunFirrtlCompilerAnnotation
```

Comment:
typo in firrt
links split across lines
should be chisel3.state. NoRunFirrtlCompilerAnnotation

Reply:
Fixed in rebase.